.github/workflows/main.yml

```diff
@@ -0,0 +1,57 @@
+name: Main branch workflow
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
This is the same behavior as CI's for all of our charms (ex. [here](https://github.com/canonical/sdcore-amf-k8s-operator/blob/main/.github/workflows/main.yaml)). We run the CI on push to main and PR to main

Reply:
[Here](https://github.com/canonical/sdcore-upf-k8s-operator/blob/main/.github/workflows/main.yaml) it's on every push :) 
All I'm saying is that it can be useful to have CI running on every push. You can get a feedback on your changes without creating a PR. Personally, I'm not a fan of work-in-progress PRs with failing tests. I never know whether such PR is ready to review or not. Running CI on every push gives me opportunity to test my changes before creating the PR. 

In this case it's probably not super important, because we don't have any tests here. I'm just using the occasion to discuss this; in case someone, some day, decides to copy-paste this config to another project ;)

Reply:
I updated the CI to run on every push